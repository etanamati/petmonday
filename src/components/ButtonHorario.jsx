import {Button} from 'react-bootstrap';
import React from 'react';
import styled from 'styled-components';

const StyledButton = styled(Button)`
  width: 25%;
  margin-top: 5px;
`;

const ButtonHorario = ({handleClick, hora, agendado}) => {
  return (
    <StyledButton variant={agendado ? "secundary" :"primary"} onClick={() => handleClick(hora)}>{hora.hora}</StyledButton>
  )
}

export default ButtonHorario;