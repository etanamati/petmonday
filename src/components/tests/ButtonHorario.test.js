import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ButtonHorario from '../ButtonHorario';

it('should render app', () => {
  const hora = {status: 'disponivel', hora: '09:00'}
  const wrapper = shallow(<ButtonHorario hora={hora}/>);
    
  expect(toJson(wrapper)).toMatchSnapshot();
});