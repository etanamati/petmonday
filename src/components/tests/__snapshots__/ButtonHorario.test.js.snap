// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render app 1`] = `
<Styled(Component)
  onClick={[Function]}
  variant="secondary"
>
  09:00
</Styled(Component)>
`;
